packages/components/statistic/src/statistic.vue

```diff
@@ -53,12 +53,15 @@ const disposeValue = computed(() => {
         .slice(1)}`
       decimal = decimal.slice(0, props.precision)
     }
-    integer = regroup(integer, props.rate, props.groupSeparator)
+    integer = groupFormat(integer, 3, props.groupSeparator)
```

Comment:
## Suggestion

How about extracting it as a variable to clarify the meaning of 3?

Reply:
You're right.